public/maps.js

```diff
@@ -171,3 +191,77 @@ fetch("https://radiant-plateau-81585.herokuapp.com/data/coverage-2018.csv")
     draw('container-map-3g-2018', "% of Population Covered by Country", data3G, "3G", 2018, { primaryColor: "#9c27b0", accentColor: "#6a0080" });
     draw('container-map-4g-2018', "% of Population Covered by Country", data4G, "4G", 2018, { primaryColor: "#4caf50", accentColor: "#087f23" });
   });
+
+
+// cities map
+const drawLatLongs = function(idName, title, dataSet, gen, year, { primaryColor, accentColor }) {
+  console.log(dataSet);
+  Highcharts.mapChart(idName, {
+    chart: {
+      map: "custom/world"
+    },
+
+    title: {
+      text: `${title}: ${gen} in ${year}`
+    },
+
+    series: [{
+        mapData: Highcharts.maps['custom/world'],
+        name: 'Basemap',
+        borderColor: '#A0A0A0',
+        nullColor: 'rgba(200, 200, 200, 0.3)',
+        showInLegend: false
+      },{
+      type: 'mappoint',
+      name: title,
+      data: dataSet,
+      name: gen,
+    }],
+
+    tooltip: {
+      headerFormat: "",
+      pointFormat: "{point.name}:</b><br>Lat:{point.lat}, Lon: {point.lon}",
+      nullFormat: "Unavailable"
+    }
+  });
+};
+
+let data = [];
+const p1 = fetch(`${dataPath}coverage-5G-2020.csv`)
+  .then(response => response.text())
+  .then((text) => {
+    const cities = text.split("\n").map((city) => city.split(","));
+    cities.map(c => {
+      if (c[0] !== "") {
+        data = data.concat( { name: c[0], lat: parseFloat(c[1]), lon: parseFloat(c[2]) } );
+      }
+    });
+  });
+
+const p2 = fetch(`${dataPath}coverage-5G-US.csv`)
+  .then(response => response.text())
+  .then((text) => {
+    const cities = text.split("\n").map((city) => city.split(","));
+    cities.map(c => {
+      if (c[0] !== "") {
+        data.push( { name: c[0], lat: parseFloat(c[1]), lon: parseFloat(c[2]) } );
+      }
+    });
+  });
+
+const p3 = fetch(`${dataPath}coverage-5G-US-expansion.csv`)
+  .then(response => response.text())
+  .then((text) => {
+    const cities = text.split("\n").map((city) => city.split(","));
+    cities.map(c => {
+      if (c[0] !== "") {
+        data.push( { name: c[0], lat: parseFloat(c[1]), lon: parseFloat(c[2]) } );
+      }
+    });
+  });
+
+Promise.all([p1, p2, p3])
+    .then((x) => {
+      drawLatLongs('container-map-2g-2014', "Cities", data, "5G", 2020, { primaryColor: "#3f51b5", accentColor:  "#002984" });
```

Comment:
For some reason `container-map-5g` wasn't working, not sure why. I switched it to this other one to test.